test/Chainweb/Test/Utils.hs

```diff
@@ -1160,4 +1160,5 @@ testRetryPolicy = stepped <> limitRetries 150
       0 -> Just 20_000
       1 -> Just 50_000
       2 -> Just 100_000
-      _ -> Just 250_000
```

Comment:
why this change?

Reply:
I was hoping this would make the tests that wait for a cut height slightly less noisy.

Reply:
I'm just going to revert this in favor of #1770 which makes this instant.